sessionctx/variable/varsutil_test.go

```diff
@@ -597,6 +596,12 @@ func (s *testVarsutilSuite) TestValidate(c *C) {
 		{TiDBEnableAmendPessimisticTxn, "0", false},
 		{TiDBEnableAmendPessimisticTxn, "1", false},
 		{TiDBEnableAmendPessimisticTxn, "256", true},
+		{TiDBAllowFallbackToTiKV, "", false},
+		{TiDBAllowFallbackToTiKV, "tiflash", false},
+		{TiDBAllowFallbackToTiKV, "  tiflash  ", false},
```

Comment:
its value should be designed to be `true`, just as `tidb_isolation_read_engines`

Reply:
The third value in curly braces is `error` field of struct, which indicates whether error would occur when executing `ValidateSetSystemVar`. `""`, `"tiflash"` and `"  tiflash  "` are valid so `error` field should be `false`.